torch/_torch_docs.py

```diff
@@ -619,6 +619,105 @@ def merge_dicts(*dicts):
     True
 """)
 
+add_docstr(torch.all,
+           r"""
+all() -> BoolTensor
+
+Returns True if all elements in the tensor are True, False otherwise.
```

Comment:
What about input tensors of dtype other than True? Maybe something like "Returns \`True\` if all elements in :attr:\`input\` evaluate to \`True\`, \`False\` otherwise."